### PR TITLE
Update README and remove outdated files

### PR DESCRIPTION
**Pull request by jonathancesario**
### Changes
- Remove all outdated files related to project creation that still use synchronous mutation. The async version should be enough.
- Restructure the README

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,74 +1,55 @@
...
+  - `client_id` and `client_secret`: generated from OAuth credentials, see [here](https://datasaurai.gitbook.io/datasaur/advanced/apis-docs/oauth-2.0).
 
-## Get Projects
+### Create Project
 
-Get projects query returns a paginated response. The input variables for the pagination can be configured directly in `get_projects.json`.
-Refer to https://api-docs.datasaur.ai/#definition-GetProjectsFilterInput for available filters.
+- The detailed explanation can be accessed [here](https://datasaurai.gitbook.io/datasaur/advanced/apis-docs/create-new-project).
+- Note that the process is asynchronous.
+- This is a multipart request because it needs to handle a list of files that will be used when creating the project.
```
Comment (tonoman3g) [inline readme.md:27]: @jonathancesario http post multipart request

File: readme.md [hunk 2]
```diff
@@ -1,74 +1,55 @@
...
-- PUT ["Tag2"]: Project A will have only Tag2.
-- PATCH ["Tag2"]: Project A will have Tag1 and Tag2.
+### Update Project Tags
 
-```
-python update_project_tags.py https://app.datasaur.ai <client_id> <client_secret> <team_id> <method>
-python update_project_tags.py https://app.datasaur.ai <client_id> <client_secret> <team_id> PUT
-```
+- Tag names and project ID can be specified from `update_project_tags.json`.
+- Two available methods: PUT and PATCH.
```
Comment (tonoman3g) [inline readme.md:49]: eh, do we have these methods?

🤔

Comment (jonathancesario) [reply readme.md:49]: the methods are introduced when creating the update project tags script based on Robosaur capability
the API doesn't have the functionality though

Comment (tonoman3g) [reply readme.md:49]: ow right got it

**Review by tonoman3g — APPROVED** [review]

**Pull request merged**